crates/sui-kvstore/src/bigtable/client.rs

```diff
@@ -237,15 +254,7 @@ impl KeyValueStoreReader for BigTableClient {
     }
 
     async fn get_latest_checkpoint(&mut self) -> Result<CheckpointSequenceNumber> {
-        let upper_limit = u64::MAX.to_be_bytes().to_vec();
-        match self
-            .reversed_scan(CHECKPOINTS_TABLE, upper_limit)
-            .await?
-            .pop()
-        {
-            Some((key_bytes, _)) => Ok(u64::from_be_bytes(key_bytes.as_slice().try_into()?)),
-            None => Ok(0),
-        }
+        self.get_watermark(AGGREGATED_WATERMARK_NAME).await
```

Comment:
Can you explain what "AGGREGATED_WATERMARK_NAME" is and why we wouldn't use the checkpoint task name? Or do we just have 1 "task" that is responsible for all the bigtable writes?

Reply:
yeah, we have a single task for all BigTable writes. 
On the sui-data-ingestion side, DDB will be eventually replaced with BigTable, so our instance will store multiple entries for different workflows (e.g., archival, blob storage, etc.). 
For community deployments, however, there will be only a single entry